### PR TITLE
471 document update

### PR DESCRIPTION
**Pull request by msevestre**

### CODE REVIEW

File: R/ospsuite-env.R [hunk 1]
```diff
@@ -1,3 +1,11 @@
+
+.getSuiteVersion <- function() {
+  version <- getNamespaceVersion("ospsuite")
```
Comment (msevestre) [inline R/ospsuite-env.R:3]: @Yuri05 @PavelBal This is in fact the better way apparently to load the version. Otheriwse, if you have two version installed, the proposed packageVersion reads the first one all the time. This reads the one loaded

File: R/ospsuite-env.R [hunk 2]
```diff
@@ -7,7 +15,7 @@ ospsuiteEnv$packageName <- "ospsuite"
 
 ospsuiteEnv$suiteName <- "Open Systems Pharmacology"
 
-ospsuiteEnv$packageVersion <- "10.0"
+ospsuiteEnv$suiteVersion <- .getSuiteVersion();
```
Comment (msevestre) [inline R/ospsuite-env.R:18]: I have renamed this to be more idiomnatic

**Comment by msevestre** [comment]
ALso I ran the styler on the package as some file were out  of style :)

File: R/ospsuite-env.R (hunk 1 above)
Comment (PavelBal) [reply R/ospsuite-env.R:3]: Looks good!

**Pull request merged**